west.yml

```diff
@@ -88,7 +88,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 3b45fbaae29640ced4eafdd9e63fe534ca1401ca
+      revision: 1b744356d7ebe88e7387ead7b52a7d96d929528d
```

Comment:
This should be: `pull/103/head` until the hal PR gets merged. See https://docs.zephyrproject.org/latest/guides/modules.html#changes-to-existing-module for details

Reply:
Done

Reply:
The HAL PR has been merged, please update this

Reply:
@mmahadevan108 Done